### PR TITLE
Adding OpenAI key to Helm Chart

### PR DESCRIPTION
**Pull request by macaab26**
## What's changing
I'm adding support for OpenAI to the Lumigator Helm chart

## How to test it
helm template test . -f values.yaml

## Related Jira Ticket
https://mzai.atlassian.net/browse/LUMI-20

## I already...

- [ ] added some tests for any new functionality;
- [ ] updated the documentation.

### CODE REVIEW

File: lumigator/infra/mzai/helm/lumigator/templates/mistral-secret.yaml [hunk 1]
```diff
@@ -1,15 +1,20 @@
-{{- include "lumigator.mistral-default-secret" . -}}
```
Comment (chainlink) [inline lumigator/infra/mzai/helm/lumigator/templates/mistral-secret.yaml:1]: rename this file

File: lumigator/infra/mzai/helm/lumigator/templates/mistral-secret.yaml [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
-  name: {{ include "lumigator.mistral-secret-name" . }}
+  name: {{ include "lumigator.lumigator-secret-name" . }}
   labels:
     {{- include "lumigator.labels" . | nindent 4 }}
 type: Opaque
 data:
+{{- if .Values.mistralAPIKey -}}
   {{ .Consts.mistralSecretKey }}: {{ .Values.mistralAPIKey | b64enc | quote }}
+{{- end -}}
+{{- if .Values.openaiAPIKey -}}
```
Comment (chainlink) [inline lumigator/infra/mzai/helm/lumigator/templates/mistral-secret.yaml:17]: add openaiAPIKey to reademe

**Review by chainlink — APPROVED** [review]
couple small changes but otherwise looks good

**Review by agpituk — APPROVED** [review]

File: lumigator/infra/mzai/helm/lumigator/templates/mistral-secret.yaml (hunk 2 above)
Comment (macaab26) [reply lumigator/infra/mzai/helm/lumigator/templates/mistral-secret.yaml:17]: It's already there! https://github.com/mozilla-ai/lumigator/pull/246/files/e67314bbd13b1d1ffa3ada0320607fe43f24aa9a#diff-d2cd8afe06d80614f32f5ff5afcd1f2cbb81154ae506f51cbc38babe4a903718R40

File: lumigator/infra/mzai/helm/lumigator/templates/mistral-secret.yaml (hunk 1 above)
Comment (macaab26) [reply lumigator/infra/mzai/helm/lumigator/templates/mistral-secret.yaml:1]: Yeah, you're right. Instead, I'm going to create a new version of the code with two different secrets (one per each key) in order to provide more flexibility to the user, as we discuss yesterday in the infra meeting.

**Pull request merged**